src/apprunner/commands/createService.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as telemetry from '../../shared/telemetry/telemetry'
+import { AppRunnerNode } from '../explorer/apprunnerNode'
+import { CreateAppRunnerServiceWizard } from '../wizards/apprunnerCreateServiceWizard'
+
+export async function createAppRunnerService(node: AppRunnerNode): Promise<void> {
+    let telemetryResult: telemetry.Result = 'Failed'
+    let source: telemetry.AppRunnerServiceSource | undefined = undefined
+
+    try {
+        const wizard = new CreateAppRunnerServiceWizard(node.region)
+        const result = await wizard.run()
+        if (result === undefined) {
+            telemetryResult = 'Cancelled'
+            return
+        }
+
+        await node.createService(result)
```

Comment:
We probably don't want logic to live in the tree nodes. Instead logic lives in the service client, and values are passed as parameters to the client functions.

The refresh() in `node.createService()` could be done as a `onAfter` callback to the client (continuation-passing-style).

Reply:
Yeah I think this brings up the question about what the nodes represent within the code. This implementation portrays them as (almost) the 'source of truth' for the corresponding resource. So any caller that wants to interact with App Runner ideally goes through the nodes, not the client. This keep the nodes in-sync as far as mutations within the toolkit go, but obviously would not scale if you had other logic dependent on the resource.

Reply:
This concept I'm describing is also at odds with current toolkit architecture. Currently, clients are easily accessible while tree nodes are not. Having the node _be_ the resource requires the inverse: nodes are easily accessible while clients are not.